src/WacVsTools.Core/AttachToWacProcess/AttachToWacProcessDialogModel.cs

```diff
@@ -20,13 +26,18 @@ public class AttachToWacProcessDialogModel : INotifyPropertyChanged
            };
 
         private DebuggerEngines debuggerEngines;
+        private WritableSettingsStore settingsStore;
 
         public AttachToWacProcessDialogModel(IMenuCommands menuCommands, IEnumerable<string> connectionTypes)
         {
             MenuCommands = menuCommands ?? throw new ArgumentNullException(nameof(menuCommands));
             ConnectionTypes = connectionTypes != null ? new ObservableCollection<string>(connectionTypes) : throw new ArgumentNullException(nameof(connectionTypes));
             Processes = new ObservableCollection<WacProcessInfo>();
             DebuggerEngines = DebuggerEngines.DefaultLazy;
+
+            var settingsManager = new ShellSettingsManager(ServiceProvider.GlobalProvider);
+            settingsStore = settingsManager.GetWritableSettingsStore(SettingsScope.UserSettings);
+            settingsStore.CreateCollection(CollectionPath);
```

Comment:
What is this going to do when it already exist?

Reply:
I'm assuming it will do nothing, but can you add a comment here to make it easier to understand?

Reply:
Yes, it skips the already existing collections.

Reply:
> I'm assuming it will do nothing, but can you add a comment here to make it easier to understand?

sure, will add a comment to clarify.